launchdarkly-android-client-sdk/src/main/java/com/launchdarkly/sdk/android/ContextDataManager.java

```diff
@@ -162,21 +139,21 @@ private void initDataInternal(
             newIndex = index.updateTimestamp(contextId, System.currentTimeMillis())
                     .prune(maxCachedContexts, removedContextIds);
             index = newIndex;
-            flagsContextId = contextId;
```

Comment:
For reviewers:  Calculating a context id is fast and trivial.  Why risk it being outdated?  Removed.